### PR TITLE
Update CredentialPack to use ParsedCredential

### PR DESCRIPTION
**Pull request by cobward**

### CODE REVIEW

File: Sources/MobileSdk/CredentialPack.swift [hunk 1]
```diff
@@ -1,53 +1,78 @@
...
-           return nil
+    /// Get credentials by id.
+    public func get(credentialsIds: [Uuid]) -> [ParsedCredential] {
+        return self.credentials.filter {
+            credentialsIds.contains($0.id())
         }
     }
+
+    /// Get a credential by id.
+    public func get(credentialId: Uuid) -> ParsedCredential? {
```
Comment (rschulman) [inline Sources/MobileSdk/CredentialPack.swift:70]: I'm a bit weirded out by having two functions with the same name and just an argument name that differs by one letter, but that might also just be a Swift convention, and I don't know Swift that well.

**Review by rschulman — APPROVED** [review]
One small comment but LGTM.

File: Sources/MobileSdk/CredentialPack.swift (hunk 1 above)
Comment (cobward) [reply Sources/MobileSdk/CredentialPack.swift:70]: I think this is a fairly normal Swift convention, but maybe @todd-spruceid or @Juliano1612 can weigh in.

Comment (Juliano1612) [reply Sources/MobileSdk/CredentialPack.swift:70]: Yeah, that's common. You only can't have multiple methods with the same signature.

**Review by Juliano1612 — APPROVED** [review]
LGTM! Started this https://github.com/spruceid/mobile-sdk-ios-app/pull/20 PR to make the demo app compatible with these changes

**Pull request merged**